docs/cluster-management/developing-locally.md

```diff
@@ -248,39 +250,38 @@ git clone git@github.com:screwdriver-cd/queue-worker.git
     # Externally routable URL for the Artifact Store
     store: http://$YOUR_IP:9002
     
- redis:
-    # Host of redis cluster
-    host: 127.0.0.1
-    # Password to connect to redis cluster, if set in first step
-    password: 'c1cd-$cr3wdriver-cd'
-    # Port of redis cluster
-    port: 6379
-    # Prefix for the queue name if needed
-    # prefix: 'beta-'
-    # Flag to enable client for TLS-based communication
-    tls: false
+ queue:
+    # Configuration of the redis instance containing resque
+    redisConnection:
+        host: "127.0.0.1"
+        port: 6379
```

Comment:
should this be `9003`?

Reply:
> should this be `9003`?

Updated document on where to configure this

Reply:
I think this `6379` is the port that `redis` is running, not the `queue-service` itself, which runs on port `9003`.

Reply:
ah I see 👍